Sources/StreamChat/Utils/SystemEnvironment+UserAgent.swift

```diff
@@ -0,0 +1,53 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+#if os(iOS)
+import UIKit
+#elseif os(macOS)
+import IOKit
+#endif
+
+extension SystemEnvironment {
+    static let userAgent: String? = {
+        guard let info = Bundle.main.infoDictionary,
+              let appName = (info["CFBundleDisplayName"] ?? info[kCFBundleIdentifierKey as String]) as? String,
+              let appVersion = info[kCFBundleVersionKey as String] as? String
+        else {
+            return nil
```

Comment:
Better to return something incomplete than to return just nil

Reply:
Fixed